Packs/FeedAWS/ReleaseNotes/1_1_12.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AWS Feed
+- Removed the parameter **feedIncremental**, after fixing adding the headers *If-None-Match* and *If-Modified-Since* to requests with server version equals or higher than 6.5.0.
```

Comment:
```suggestion
- Reverted changes made in 1.1.11.
- Fixed an issue where adding the headers *If-None-Match* and *If-Modified-Since* to requests was not compatible with Cortex XSOAR with a version below 6.5.0.
```